src/pages/airport/[icao].vue

```diff
@@ -649,6 +687,8 @@ await setupDataFetch({
         }
 
         &_pilot {
+            position: absolute;
```

Comment:
This worked fine before, you sure?

Reply:
Before the pilot overlay made the map smaller when the overlay opened, with this the overlay is above the map and does not change the map size. I thought it looks better when the map size is not changed by the overlay. But it is an optional change